src/isql/InputDevices.h

```diff
@@ -130,6 +130,11 @@ inline InputDevices::InputDevices(Firebird::MemoryPool& p)
 
 inline InputDevices::~InputDevices()
 {
+	for (unsigned n = 0; n < commands.getCount(); ++n)
+		delete commands[n];
```

Comment:
```suggestion
	for (auto command : commands)
		delete command;
```

Reply:
`Command*` is an expression simple enough, I see no need for `auto` here.

Reply:
`auto` is bad for readability. It is justified if the iterator is a multilevel template but not in this case.

Reply:
I made a comment that improves the original code imo. I'll not keep arguing about this minor thing with who is know to argue with every minor things.

Reply:
Thanks for review. The only reason I chose a non range-based for loop without auto is because I wanted to keep the same code for all major versions, including FB 3.0 (that could be built with C++ 2003 on windows)

Am I wrong in such approach?